deploy/examples/create-external-cluster-resources.py

```diff
@@ -791,7 +931,21 @@ def create_rgw_admin_ops_user(self):
                 err_msg = "failed to execute command %s. Output: %s. Code: %s. Error: %s" % (
                     cmd, execErr.output, execErr.returncode, execErr.stderr)
                 raise Exception(err_msg)
-
+        
+        # ceph version should atleast quincy to validate rgw-endpoint 
```

Comment:
Actually, I just realized... There is one problem though, downstream has support for this backported and the upstream backport is still pending... So this will be skipped downstream :( 
Can we read the output error from radosgw-admin when failing with unknown caps?

Reply:
Done